src/legacy/ui/public/vis/editors/default/agg_param.js

```diff
@@ -58,6 +59,7 @@ uiModules
             value="paramValue"
             is-invalid="isInvalid"
             set-validity="setValidity"
+            field="agg.params.field"
```

Comment:
I added the `field` attribute to rerendering of  aggParams' React components when the field value is changed.